verification/src/solidity/compiler_cache.rs

```diff
@@ -0,0 +1,85 @@
+use ethers_solc::Solc;
+use semver::Version;
+use std::{collections::HashMap, sync::Arc};
+
+#[derive(Default)]
+pub struct CompilerCache {
```

Comment:
Also, some explicit initialization would be useful to allow reuse locally downloaded compilers in case the service will be rerun